concepts/40 Angular Components/10 Getting Started/90 Support for Angular Signals.md

```diff
@@ -0,0 +1,14 @@
+Angular v16 and up includes a popular capability called "Angular Signals". It is a new, streamlined way to track changes in page data, and react to these changes.
+
+**Signals** are wrappers that listen to changes in page data. You can create two kinds of signals:
+
+* A *writable signal* wraps around a variable and fires when the variable's value changes. You can reference this signal throughout the application to manually overwrite the value of the underlying variable.
+* A *computed signal* is derivative. It uses internal logic to process data from other signals.
```

Comment:
What do you mean by "derivative"? Are you using this word as a noun in a mathematical context:
"the limit of the ratio of the change in a function to the corresponding change in its independent variable as the latter change approaches zero"